src/components/InputTooltip.js

```diff
@@ -20,7 +20,11 @@ class InputTooltip extends Component {
     }
 
     componentDidMount() {
-        this.input.focus();
+        const { isEdit } = this.props;
+
+        if (!isEdit) {
```

Comment:
to be specific, if someone clicks on a link to edit, it won't be focused as well, aye?